scripts/rel_otp_apps.eterm

```diff
@@ -0,0 +1,16 @@
+%% Single source of truth of list otp apps that we use
+[ kernel
+, stdlib
+, sasl
+, crypto
+, public_key
+, asn1
+, syntax_tools
+, ssl
+, os_mon
+, inets
+, compiler
+, runtime_tools
+, {mnesia, load}
+, xmerl
```

Comment:
just a note: xmerl is now included in edge edition, the reason why it was not included in edge edition is unknown. this change seems to be fine.